src/pages/HomePage/CreatePostBox/CreatePostBox.jsx

```diff
@@ -1,11 +1,25 @@
 import React, { useState, useEffect } from "react";
 import "./CreatePostBox.css";
 import TextareaAutosize from "react-textarea-autosize";
-import { createNewPost, editPost } from "../../../features/posts/postsSlice";
+import {
+  createNewPost,
+  editPost,
+  addPostToDraft,
+} from "../../../features/posts/postsSlice";
 import { useDispatch, useSelector } from "react-redux";
+import { CreatePollModal, PollBody } from "../../../components/index";
 
 function CreatePostBox({ isPostEdit, setIsPostEdit, editPostId }) {
   const [postData, setPostData] = useState({ content: "" });
+  const [pollModal, setPollModal] = useState(false);
+  const [pollData, setPollData] = useState({
+    question: "",
+    option1: "",
+    option2: "",
+    option3: "",
+    option4: "",
+    showPoll: false,
+  });
```

Comment:
extract it in another variable so that you don't need to write it twice as you did it in line 48
`const init = {
    question: "",
    option1: "",
    option2: "",
    option3: "",
    option4: "",
    showPoll: false,
  }`

Reply:
Yaah make sense